addons/hubble/hubble.yaml

```diff
@@ -0,0 +1,517 @@
+# Copyright 2021 The Kubermatic Kubernetes Platform contributors.
```

Comment:
Do you wanna deploy this, even if Cilium is not used ?

Reply:
Nope, that does not make sense. Will protect it in the manifest template.